app/person-escort-record/controllers/print-record.js

```diff
@@ -107,6 +109,19 @@ function printRecord(req, res) {
       )
       .map(response => response.value)
   })
+  const handoverDetails = {
```

Comment:
Do we need any tests to ensure these values are outputted correctly?

Reply:
There is unfortunately a little gap in this controller at the moment. This whole part could do with some better refactoring but I didn't want to end up down the rabbit hole just yet...

Reply:
I guess that can be the work of a future PR